CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.16.1 (2020-03-18)
+
+- Fixed redirecting the empty-path (root) route. [#230](https://github.com/blackbaud/skyux-sdk-builder/pull/230)
```

Comment:
`Fixed redirecting` sounds weird to me. Would it work to change this to `Fixed the redirect for the empty-path (root) route.`

Reply:
Sounds good.  I'll update now.